### ISSUE TITLE
Howler - Player not working properly on mobile

### ISSUE THREAD

**Issue by malinga91** [goldfire/howler.js#922]
I need to play few mp3 files one after another. I used have used howler **Player** it is working properly on desktop. But not on mobile. Its only plays first mp3 file only and others not playing. Need to click end of each audio. I tried to get permission as follow. But not luck
```
document.addEventListener('touchstart', function(e) {
        e.preventDefault();
        var muteSound = new Howl({
            src: ['app/beats/E.mp3'],
			mute: false
        });
        muteSound.mobileAutoEnable = false;
        muteSound.play();
    }, false);
```

**Comment by Nuuf** [comment]
The documentation states: `Howler.mobileAutoEnable`

**Comment by Nuuf** [comment]
Also, you should convert to multiple file types. ogg, wav, etc. Some systems may not support a specific one.

**Comment by Ziphoid** [comment]
To most certainty, this is because you're using HTML5 audio and not the Web Audio API. Set `html5: false` and you should be golden. For some reason, it seems like it's not setting it to false as default even though the documentation states that, so you'll have to enforce it. The bad side of this is that you won't be able to stream the track and the browser will have to download it fully before it starts playing, but on the other hand, you're fully able to do audio processing for making visualization and whatnots. :)

**Comment by Mixddux** [comment]
I changed `html5: false` but that alone didn't seem to do it, so I removed the .webm from the src and I was able to get it working. Of course a slight delay when starting, but at least it's working! Thanks!

```
src: ['./audio/' + data.file + '.mp3'],
        // src: ['./audio/' + data.file + '.webm', './audio/' + data.file + '.mp3'],
        html5: false, // Force to HTML5 so that the audio can stream in (best for large files).
```

**Comment by goldfire** [comment]
This should now be fixed in v2.1.0 using the HTML5 Audio pool.

**Comment by gio-shara-code** [comment]
I know its closed but if there is someone in 2024 this is the solution:

1. Remove `.webm` file from `src` if you have one
2. Set `html5: true`

example:
```typescript
    const howl = new Howl({
      src: ['./audios/game-music.mp3'],
      volume: BACKGROUND_MUSIC_VOLUME,
      html5: true,
      format: ['mp3', 'webm'],
      loop: true,
    });
```

Mobile Safari and Chrome both works.
